kt/entry-generation/godot-entry-generator/src/main/kotlin/godot/entrygenerator/filebuilder/ClassRegistrarFileBuilder.kt

```diff
@@ -56,11 +50,14 @@ class ClassRegistrarFileBuilder(
         .beginControlFlow("with(registry)") //START: with registry
         .let { funSpecBuilder ->
             if (!registeredClass.isAbstract) {
+                val superClasses = registeredClass.supertypes.takeWhile {
+                    //Used to implement script inheritance methods, so we remove base types and abstract parents.
+                    it.name != registeredClass.godotBaseClass && !it.isAbstract
+                }.map { "\"${it.name}\"" }.reduceOrNull{ acc, string -> "$acc,$string" }?:""
```

Comment:
`acc`? The time you "gained" by not typing the remaining characters is time i waste every time trying to guess what that name means 🤣